cmd/operate/env.go

```diff
@@ -1 +1,96 @@
 package operate
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/dream11/odin/internal/service"
+	"github.com/dream11/odin/pkg/constant"
+	environment "github.com/dream11/odin/proto/gen/go/dream11/od/environment/v1"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"google.golang.org/protobuf/types/known/structpb"
+)
+
+var envName string
+var data string
+var operateEnvClient = service.Environment{}
+
+var operateEnvCmd = &cobra.Command{
```

Comment:
do we need operations on env?

Reply:
have only added an empty command body. Implementation is left blank. This is just to speed up development when we decide to implement operate in future